js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -1074,6 +1075,8 @@ var EditingController = modules.ViewController.inherit((function() {
             if(deferreds.length) {
                 that._saving = true;
 
+                dataSource && dataSource._changeLoadingCount(1);
```

Comment:
Interesting:
- https://github.com/DevExpress/DevExtreme/blob/17_2/js/ui/pivot_grid/ui.pivot_grid.data_controller.js#L906-L911
- https://github.com/DevExpress/DevExtreme/blob/17_2/js/ui/data_grid/ui.data_grid.grouping.js#L99-L100

I vote for this:

> add beginLoading/endLoading methods to prevent using private API.